diff --git a/dist/client/getDeviceDescriptor.js b/dist/client/getDeviceDescriptor.js
index 666256761463ab693319f8e3b761d3ec5ee8593e..b0a6129f61a4ce823a51c84f076f1c2d8b0a8d9f 100644
--- a/dist/client/getDeviceDescriptor.js
+++ b/dist/client/getDeviceDescriptor.js
@@ -36,5 +36,13 @@ export const getDeviceDescriptor = () => {
             osVersion: '',
         };
     }
+    if (platform.OS === 'windows') {
+        return {
+            platform: 'windows',
+            manufacturer: '',
+            model: '',
+            osVersion: String(platform.constants?.osVersion ?? ''),
+        };
+    }
     throw new Error('Unsupported platform');
 };
